### PR TITLE
BANNO.ODTOPTIN.V1.POW Add ODT amount clear param

### PR DESCRIPTION
**Pull request by tkainz**
Adding parameter setting to clear/not clear the Share ODT Amount upon opting the share out of ODT

### CODE REVIEW

File: odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW [hunk 1]
```diff
@@ -48,6 +52,8 @@ DEFINE
 
  TRUE                         = 1
  FALSE                        = 0
+ YES                          = 1
```
Comment (rrobison44) [inline odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW:55]: Probably doesn't matter much but just wondering on the decision to use YES/NO vs. just having a TRUE/FALSE setting.

**Review by rrobison44 — APPROVED** [review]

File: odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW (hunk 1 above)
Comment (tkainz) [reply odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW:55]: @rrobison44 Just seemed like a more proper way to respond to the CFG question **"When a member opts-out of OTA should the Share's OVERDRAFTTOLERANCE field be set to $0.00?"**

Comment (tkainz) [reply odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW:55]: @rrobison44 If you'd like it changes to TRUE/FALSE - no problems - just let me know. It'll work the same either way!

**Review by jordan-keenan — APPROVED** [review]
Hi @tkainz, Code Changes look good!

**Review by jucarson — APPROVED** [review]
Looks good Tom!

File: odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW (hunk 1 above)
Comment (tkainz) [reply odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW]: @rrobison44 Reverted 'YES'/'NO' booleans to 'TRUE'/'FALSE' in the PowerOn code

**Pull request merged**